data/language/en-US.txt

```diff
@@ -924,6 +924,11 @@ STR_6206    :Gray stucco
 STR_6212    :Gray sandstone
 STR_6274    :Can't set color scheme...
 
+STR_6308    :Scroll wheel behavior:
```

Comment:
You should only change the en-UK file on develop. All other language files are updated through our Localisation repository. (If we'd change them here, that would lead to a conflict.)